backend/db/company_jobs.py

```diff
@@ -0,0 +1,18 @@
+from sqlmodel import SQLModel, Field, UniqueConstraint
+from datetime import datetime
+
+class Company_jobs(SQLModel, table=True):
```

Comment:
```suggestion
class CompanyJobs(SQLModel, table=True):
```
https://google.github.io/styleguide/pyguide.html#:~:text=Use%20CapWords%20for%20class%20names%2C

Reply:
I've made changes to the code in the branch.